TYPO3.Flow/Classes/TYPO3/Flow/Configuration/ConfigurationManager.php

```diff
@@ -651,24 +651,51 @@ protected function postProcessConfiguration(array &$configurations)
         foreach ($configurations as $key => $configuration) {
             if (is_array($configuration)) {
                 $this->postProcessConfiguration($configurations[$key]);
```

Comment:
Oh, just realize whenever we want to do that, this has a couple of implications:
We would need to take care of (correct) string quoting and we need to figure out a way to understand if a match is the complete assignment and take care of the variable type thingy. So all in all that will not be that easy.